blob/azure-storage-blob.gemspec

```diff
@@ -38,10 +38,10 @@ Gem::Specification.new do |s|
   s.license     = "MIT"
   s.files       = `git ls-files ./lib/azure/storage/blob/`.split("\n") << "./lib/azure/storage/blob.rb"
 
-  s.required_ruby_version = ">= 1.9.3"
+  s.required_ruby_version = ">= 2.4.0"
 
   s.add_runtime_dependency("azure-core",              "~> 0.1.13")
-  s.add_runtime_dependency("azure-storage-common",    "~> 1.0")
+  s.add_runtime_dependency("azure-storage-common",    "~> 1.2")
```

Comment:
Because this is breaking change, common will have to bump up to 2.0.0 among all other services.

Reply:
Bumped versions to 2.0.0 in e82f9e8.